File: conditionals/notes.py
# Elizabeth Gutierrez, Conditionals notes Python
name = input("What is your name?:\n")
#1. What puts something inside of the “if” statement?
# Having a tab/white space makes something inside of an if statement.
<<<<<<< HEAD
if name == "LaRose":
    print("Hi Ms. LaRose")
else: # happens if condition is false.
    print(f"Hello {name}!")

=======
if name == "Elizabeth":
    print("Elizabeth")
else: 
        print(f"Hello {name}!")
#happens if condition is false.
>>>>>>> 0d9360a0195fbc64745ea54fe853d60317c88bf0
#2. What do if statements do?
# if statements checks a condition and it it is true, it will do a thing.
if name == "Ms. LaRose": # <= this is the condition.
    print("Hi Mr. LaRose") # <= What it does if true.

#3. What are boolean statements? 
# A boolean statement is the part of conditional that is either true or false(true or false statement).
<<<<<<< HEAD
if name == "LaRose": # <- This is the boolean statement(either true or false).
    print("Hi Ms. LaRose")
else: # happens if condition os false.
    print(f"Hello {name}!")
=======
if name == "Elizabeth": # <- This is the boolean statement(either true or false).
    print("Elizabeth")
else: 
        print(f"Hello {name}!")
>>>>>>> 0d9360a0195fbc64745ea54fe853d60317c88bf0

#4. What do else statements do?
if name == "LaRose":
    print("Hi Ms. LaRose")
else: #if boolean is false, the else statement happens.
    print(f"Hello {name}!")
# else statements shows if boolean statement is false.

#5. What kind of statement do you use if you have more than 2 needed outcomes?
num = int(input("How many cookies are there?\n"))
#computers read top to bottom, check the leat likely first(more specific on top, less specific on bottom).
if num == 0: # <- if always starts the conditional
    print("There are none.")
elif num == 1: # <- everything in between should be elif.
    print("There is one.")
elif num <4: # <- everything in between should be elif.
    print("There are a couple")
elif num < 10: # <- everything in between should be elif.
    print("There are a few")
else: # <- always ends the conditional
    print("There are many")
#6. What do each of the different symbols mean in conditionals?
#< This means less than
#> This means greater than
#<= This means less than or equal to
#>= This means greater than or equal to
#== This means equal to
#=== Doesn't exist in python(only in C)
#! This means not
#7. What are the 3 logical operators?
<<<<<<< HEAD
if num < 10 and num > 5: #And both booleans must be true.
      print("This is a big single digit number")
elif num < 10 or num > 5: #or means one must be true.
      print("This is a big single digit number")
elif not num < 10: #Not changes it to check if false.
      print("This is a big single digit number")
=======
elif num < 10 and num > 5: #And both booleans must be true.
    print("This is a big single digit number")

elif num < 10 or num > 5: #or means one must be true.
    print("This is a big single digit number")

elif not num < 10: #Not changes it to check if false.
    print("This is not a single digit number")
>>>>>>> 0d9360a0195fbc64745ea54fe853d60317c88bf0

#8. What are logical operators for?
#Allows the code to handle more difficult question, increases complexity.

#9. What does a nested conditional statement do?
if num < 10:
    if num ==8:
        print("This prints at 8")
    else:
        if num == 4:
<<<<<<< HEAD
            print("There are only enough cookies left for me")
        else:
            print("The number is less than 10")
=======
            print("There are only enough cookies left for me. . . sorry")
        else:
        print("The number is less than 10")
>>>>>>> 0d9360a0195fbc64745ea54fe853d60317c88bf0
else:
    print("The number is bigger than 10")

# you can nest as many conditionals as you want, but stop at three to not confuse it.

File: conditionals/notes.c
// Elizabeth Gutierre, Conditionals notes C
#include <stdio.h>
#include <string.h> //needed to compare strings
char name[]= "Elizabeth";
int num;

int main(void){
    //10. How do you write an if statement in C?
    if(strcmp(name, "Elizabeth") == 0){ //strcmp measn string comparison when the outcome is 0 the string are the same.

        printf("Hello Elizabeth, welcome to my class.\n");
    //11. How do you write an else statement in C?
    }else{
        printf("Hello %s, welcome to my class.\n", name);
    }
    printf("How many siblings do you have?\n"); //for practice assignment
    scanf("%d", &num);
    //12. How do you write elif/ else if statements in C?
    if(num == 0){
    printf("You are an only child\n");
    }else if(num <= 2){
        printf("You have a couple siblings\n");
    }else if(num <= 5){
        printf("You have a few siblings\n");
    }else{
        printf("Your have a lot of sbilings\n");
    }

   //13. How do you write the 3 logical operators in C?
   // && = and
   // || = or
   // ! = not

   if(num == 7 ||  num == 13){
      printf("%d is an unlucky number\n", num);
    }else if (num <10 && num <5){
      printf("%d is a large single digit number\n", num);
   }else if(!(num > 10)){
       if(num == 12){
           printf("That is a dozen!\n");
       }else{
           printf("%d is a big number\n", num);
       }
   }else{
      printf("You typed in %d\n", num);
   }
   return 0;
}
